src/frecent_paths.rs

```diff
@@ -68,9 +67,8 @@ impl PathFrecency {
         self.frecency
             .serialize(&mut rmp_serde::Serializer::new(tmpfile))
             .map_err(|_| "could not create tmpfile".to_string())?;
-        fs::rename(tmpfile_path, &self.path).map_err(|e| {
-            format!("could not atomically rename: {}", e).to_string()
-        })
+        fs::rename(tmpfile_path, &self.path)
+            .map_err(|e| format!("could not atomically rename: {}", e).to_string())
```

Comment:
Isn't it more kosher for this function to return Result<(), Error> (or whatever the rustism for that is)

Reply:
Yes, but since `Error` is just a trait, I'd have to pick a concrete type (e.g. String) and then implement Error for it.

Since I'm not yet matching on errors or doing anything more than printing them to users, I don't see the need for adding that extra boilerplate vs just doing effectively golang-style error handling for now.

If this were a lib I'd definitely want to do the right thing, but as a internal-to-this-bin bit of code, I don't think it's a big deal to do this for now.